### DISCUSSION TITLE
`defineEnvironment` fails to define `app.timezone`

### DISCUSSION THREAD

**Discussion by devajmeireles** [orchestral/testbench#415] [Q&A] [answered]
- Testbench Version: 9.3.0
- Testbench Dusk: 9.6.0
- Laravel Version: 11.21.0
- PHP Version: 8.3.10
- Database Driver & Version: N/A

### Description:

I'm working on TallStackUI V2 - rewriting some tests and improving our test suite with new methods. 

Today, I started working on the Date component and when I tried to set the timezone to `America/São_Paulo` using `defineEnvironment` I'm having problems. It doesn't seem to work. I can only edit the timezone via `getApplicationTimezone`. Should the timezone only be set using `getApplicationTimezone`? 

I'm questioning this because the docs says: "**You can also** easily override the application's default timezone" - this makes it seem like there is another way.

![CleanShot 2024-08-25 at 22 38 45](https://github.com/user-attachments/assets/77aa6893-0714-4ced-9cbb-993c62877c9d)

Result:
![CleanShot 2024-08-25 at 22 45 30](https://github.com/user-attachments/assets/2aa8febb-e1a3-46dc-8cae-b103ccd7bd47)

### Steps To Reproduce:

Try to set the `app.timezone` via `defineEnvironment` and check if it will be applied.

**Comment by crynobone** [comment, marked as answer]
> Should the timezone only be set using getApplicationTimezone?

`getApplicationTimezone()` also set `date_default_timezone_set()` similar to `Illuminate\Foundation\Bootstrap\LoadConfiguration` which does happen much earlier than `defineEnvironment()`. So yes I believe this is the correct way to set timezone.
